api/src/entities/mint_histories.rs

```diff
@@ -1,34 +1,30 @@
-//! `SeaORM` Entity. Generated by sea-orm-codegen 0.11.0
+//! `SeaORM` Entity. Generated by sea-orm-codegen 0.11.3
 
 use async_graphql::SimpleObject;
 use sea_orm::entity::prelude::*;
 
 use super::sea_orm_active_enums::CreationStatus;
 
 #[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq, SimpleObject)]
-#[sea_orm(table_name = "purchases")]
-#[graphql(concrete(name = "Purchase", params()))]
-/// Represents the purchase of an NFT.
+#[sea_orm(table_name = "mint_histories")]
+#[graphql(concrete(name = "MintHistories", params()))]
 pub struct Model {
-    /// The ID of the purchase.
     #[sea_orm(primary_key, auto_increment = false)]
     pub id: Uuid,
     /// The ID of the NFT being purchased.
     pub mint_id: Uuid,
-    /// The ID of the drop that facilitated the purchase, if any.
-    pub drop_id: Option<Uuid>,
     /// The wallet address of the buyer.
     #[sea_orm(column_type = "Text")]
     pub wallet: String,
-    /// The amount spent on the purchase.
-    pub spent: i64,
     /// The signature of the transaction, if any.
     #[sea_orm(column_type = "Text", nullable)]
     pub tx_signature: Option<String>,
     /// The status of the creation of the NFT.
     pub status: CreationStatus,
     /// The date and time when the purchase was created.
     pub created_at: DateTimeWithTimeZone,
+    /// The ID of the collection that facilitated the purchase, if any.
+    pub collection: Option<Uuid>,
```

Comment:
There is always going to be a collection so this should be an option and should be collection_id in the case the we add a field resolve to fetch the actual collection.